test/e2e/specs/support/support__showme-where.ts

```diff
@@ -20,17 +21,20 @@ describe( 'Support: Show me where', function () {
 
 	it( 'Search for help: Create a site', async function () {
 		supportComponent = new SupportComponent( page );
-		await supportComponent.showSupportCard();
+
+		await supportComponent.openPopover();
+
 		await supportComponent.search( 'create a site' );
-		const results = await supportComponent.getResults( 'article' );
-		expect( await results.count() ).toBeGreaterThan( 0 );
 	} );
 
 	it( 'Click on result under Show me where', async function () {
-		await supportComponent.clickResult( 'where', 1 );
+		[ newPage ] = await Promise.all( [
+			page.waitForEvent( 'popup' ),
+			supportComponent.clickResult( 'Create a new site' ),
+		] );
 	} );
 
 	it( 'Signup flow is started', async function () {
-		await page.waitForURL( /start/ );
+		await newPage.waitForURL( /start/ );
```

Comment:
Same as above - would be nice to have an example URL in the comment here 🙏

Reply:
Example added in 23ed24e76e93e4ec1b0598cbedc0e6ad2f89f4dd.